SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -12,6 +12,7 @@
 #include "Utils/CodeWriter.h"
 #include "Utils/ComponentIdGenerator.h"
 #include "Utils/DataTypeUtilities.h"
+#include "SpatialGDKEditorSchemaGenerator.h"
```

Comment:
I had an impression that SpatialGDKEditorSchemaGenerator is an extension that uses SchemaGenerator. If that is so, then it's not nice that the base SchemaGenerator depends on SpatialGDKEditorSchemaGenerator.

Reply:
Why can't we just put `IsSupportedClass` into SchemaGenerator?

Reply:
Let me know if you'd prefer to keep it as it is or planning to change it for this PR.